ironfish/src/telemetry/telemetry.ts

```diff
@@ -328,4 +329,39 @@ export class Telemetry {
       ],
     })
   }
+
+  submitNewTransactionSeen(transaction: Transaction, seenAt: Date): void {
+    if (this.shouldLogTransaction(transaction)) {
```

Comment:
we could get rid of this indentation if we just early return like the other telemetry functions

Reply:
yea I'll do this